README.md

```diff
@@ -57,6 +57,25 @@ For example, to import the Merkle Proof library use the following statement:
 sway_libs::binary_merkle_proof::verify_proof;
 ```
 
+## Running Tests
```

Comment:
Consider moving this over to the CONTRIBUTING.md as you only need it if you are forking/cloning the repo and developing on a branch.